File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.py
def solution(strArr):
    dic = {}
    for s in strArr:
        l = len(s)
        if l not in dic:
            dic[l] = 1
        else:
            dic[l] += 1
    
    a_value = 0
    for key, value in dic.items():
        if value > a_value:
            a_value = value
            
    return a_value

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int[] arr = new int[31];
        for (String s: strArr) {
            arr[s.length()]++;
        }
        int max = arr[1];
        for (int i = 2; i < 31; i++) {
            max = Math.max(max, arr[i]);
        }
        return max;
    }
}